cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PageRankBenchmark.scala

```diff
@@ -24,4 +24,4 @@ class PageRankBenchmark(graph: DirectedGraph[Node],
   def operation() {
     PageRank(graph, pageRankParams)
   }
-}
\ No newline at end of file
+}
```

Comment:
Just added cassovary-benchmarks/BenchmarksREADME.md -- please feel free to
modify and look at the individual sources of benchmarks as well.

On Thu, Mar 5, 2015 at 2:58 PM, Ben McKown <notifications@github.com> wrote:

> In
> cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PageRankBenchmark.scala
> <https://github.com/twitter/cassovary/pull/167#discussion_r25910667>:
>
> > \ No newline at end of file
> > +}
>
> After trying to figure out how to run benchmarks for a while, I decided to
> just ask. How do I run the benchmarks? Thanks!
>
> I will probably update the README with instructions on how to run them,
> unless of course I'm just being an idiot.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/cassovary/pull/167/files#r25910667>.
>